internal/pkg/template/workload.go

```diff
@@ -282,6 +282,30 @@ func ImportedVariable(name string) Variable {
 	return importedEnvVar(name)
 }
 
+// GetAliases return all the unique aliases specified across all the routing rules in NLB.
+// Currently, we only have primary routing rule, but we will be getting additional routing rule soon.
+func (cfg *NetworkLoadBalancer) GetAliases() []string {
```

Comment:
nit: 👀 Check out the first bullet point here: https://google.github.io/styleguide/go/best-practices#naming-conventions